docs/quickstart.rst

```diff
@@ -11,6 +11,14 @@ Install a package from `PyPI`_:
     [...]
     Successfully installed SomePackage
 
+Install a package downloaded from `PyPI`_ or got elsewhere. This is useful if the target machine does not have a network connection:
+
+::
+
+  $ pip install SomePackage.tar.gz
```

Comment:
Hmm, it seems I need to recreate this PR anyway. I deleted my very old pip fork without realizing that this PR that I did via the web UI was also created based on that. I'll wait for possible further comments for a little time first.